app/uk/gov/hmrc/plasticpackagingtax/registration/views/models/BackButton.scala

```diff
@@ -17,15 +17,9 @@
 package uk.gov.hmrc.plasticpackagingtax.registration.views.models
 
 import play.api.mvc.Call
+import uk.gov.hmrc.http.HttpVerbs.GET
 
-sealed trait BackButtonLink {
-  val title: String
-  val hiddenText: String
-  val call: Call
-}
+sealed trait BackButtonLink
 
-case class BackButton(override val title: String, override val call: Call, override val hiddenText: String) extends BackButtonLink
-
-case class BackButtonJs(override val title: String, override val hiddenText: String) extends BackButtonLink {
-  override val call: Call = Call("GET", "javascript:window.history.back()")
-}
+case class BackButton(title: String, call: Call,  hiddenText: String) extends BackButtonLink
+case class BackButtonJs(call: Call = Call(GET, "javascript:window.history.back()")) extends BackButtonLink
```

Comment:
if you want to this could be an object, as the call is always "javascript:window.history.back()"

Reply:
I see what you mean, bur the BackButtonsJS is also used by the partner_check_answer_page which pass a commonRoutes.TaskListController.displayPage() call to it. Thanks